### PR TITLE
tailwind: Add prettier plugin for sorting classes

### PR DESCRIPTION
**Pull request by SibiAkkash**
Adding prettier plugin which sorts the tailwind classes to a consistent order.

@punchagan @Joe2k shall I lint all files and push here ?

### CODE REVIEW

File: frontend/prettier.config.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/** @type {import("prettier").Config} */
```
Comment (punchagan) [inline frontend/prettier.config.js:1]: Is there a reason you want to move this to a separate file?

File: scripts/lint [hunk 2]
```diff
@@ -61,7 +61,7 @@ def run() -> None:
         [
             "frontend/node_modules/.bin/prettier",
             "--config",
-            "frontend/package.json",
+            "frontend/prettier.config.js",
```
Comment (punchagan) [inline scripts/lint:64]: Add this in a separate commit, where this change comes with all the files linted correctly.

**Review by punchagan — COMMENTED** [review]
Thanks! This is good. 

Also, adding a eslint or prettier tool to sort all imports would be good! Feel free to add that in separate commits or a separate PR.

File: frontend/prettier.config.js (hunk 1 above)
Comment (SibiAkkash) [reply frontend/prettier.config.js:1]: https://github.com/prettier/prettier/issues/15085
When i added the plugin in the `package.json` file, prettier wasn't able to find the package. I had to pass in the path to the plugin. So i moved to a separate file.

File: scripts/lint (hunk 2 above)
Comment (SibiAkkash) [reply scripts/lint:64]: yeah makes sense

**Comment by SibiAkkash** [comment]
> Thanks! This is good.
> 
> Also, adding a eslint or prettier tool to sort all imports would be good! Feel free to add that in separate commits or a separate PR.

Have added the [simple-import-sort](https://github.com/lydell/eslint-plugin-simple-import-sort#readme) plugin for eslint.
The `sort-imports` rule doesn't actually fix the lint errors, so i looked at some libraries that would autofix.  I looked at another called [eslint-plugin-import](https://github.com/import-js/eslint-plugin-import), but `simple-import-sort` looked the easiest

**Comment by punchagan** [comment]
Thanks for this! I'll merge this once we have all our UI PRs merged in, so as to prevent conflicts in them.

File: frontend/prettier.config.js (hunk 1 above)
Comment (punchagan) [reply frontend/prettier.config.js]: I changed the path to a relative one, and it seems to work.

**Pull request merged**